test/FastTests/Server/Documents/Queries/Dynamic/MapReduce/RavenDB_8076.cs

```diff
@@ -0,0 +1,214 @@
+﻿using Raven.Client.Documents.Queries;
+using Raven.Tests.Core.Utils.Entities;
+using Sparrow.Json;
+using Xunit;
+
+namespace FastTests.Server.Documents.Queries.Dynamic.MapReduce
+{
+    public class RavenDB_8076 : RavenTestBase
+    {
+        [Fact]
+        public void Can_sort_by_aggregation_function_in_dynamic_group_by_query()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (var s = store.OpenSession())
+                {
+                    s.Store(new Company
+                    {
+                        Address1 = "UK",
+                        AccountsReceivable = 1
+                    });
+
+                    s.Store(new Company
+                    {
+                        Address1 = "UK",
+                        AccountsReceivable = 1
+                    });
+
+                    s.Store(new Company
+                    {
+                        Address1 = "USA",
+                        AccountsReceivable = 1
+                    });
+
+                    s.SaveChanges();
+                }
+
+                using (var commands = store.Commands())
+                {
+                    foreach (var q in new(string Query, string Field)[]
+                    {
+                        (@"select count()
+                        from Companies
+                        group by Address1 order by count()", "Count"),
```

Comment:
Shouldn't the field name here be `count()` ? How do we determine the default field names here?